VMBackup/main/freezesnapshotter.py

```diff
@@ -159,8 +159,8 @@ def updateErrorCode(self, blob_snapshot_info_array, all_failed):
                         error_msg = error_msg + ExtensionErrorCodeHelper.ExtensionErrorCodeHelper.StatusCodeStringBuilder(self.extensionErrorCode)
                         self.logger.log(error_msg, True, 'Error')
                         break
-                    elif blob_snapshot_info.isSuccessful == False and not all_failed:
```

Comment:
this elif should still be kept

Reply:
Yes, i think i should still keep that. will do that change.